recipes/simplification/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "simplification" %}
+{% set version = "0.5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/{{ name }}/tarball/{{ version }}
+  sha256: 1ab55d160719803cecb79d6cfc6c4a488011821d49224fbe3acb08b986b2298a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37, py>38]
```

Pretty sure this build should be failing due to `numpy` no longer being listed (it's imported right at the top of `setup.py`). I think this is why:
```suggestion
  skip: True  # [py<37 or py>38]
```